src/components/TodoList.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import axios from "axios";
+import {AiOutlineEdit, AiOutlineDelete, MdOutlineDone} from "react-icons/all";
 
 const TodoList = (props = (todos, isLoading, setIsLoading)) => {
```

Comment:
What is this syntax? You are not destructuring. You are trying to assign something to props?

Reply:
I am doing this so I can use props.propname. Its how we always set it up in the previous project.

Reply:
This is not assigning anything to props. You can try to rename the variables and it wont make a difference.
```
props = (opala, test, lol)
```
This still has the same effect. When using (...) inside the brackets something get evaluated.
An example could be `props = (ENV === "dev" ? {env: "dev"} : {env: "prod", other: "random"})`

Valid syntax:
```
const Component = (props = {todos: [], isLoading: false, setIsLoading: () => {}}) => {}
```
This is valid syntax, although props is already an object, so you can safely de-structure
```
const Component = ({todos: [], isLoading: false, setIsLoading: () => {}}) => {}
```